packages/zoe/tools/fakePriceAuthority.js

```diff
@@ -162,32 +177,45 @@ export async function makeFakePriceAuthority(options) {
         } else {
           currentPriceIndex += 1;
         }
+        latestTick = t;
         tickUpdater.updateState(t);
         for (const req of comparisonQueue) {
-          // eslint-disable-next-line no-await-in-loop
-          const priceQuote = priceInQuote(req.amountIn, req.brandOut, t);
-          const { amountOut: quotedOut } = priceQuote.quoteAmount.value[0];
-          if (req.operator(quotedOut)) {
-            req.resolve(priceQuote);
-            comparisonQueue.splice(comparisonQueue.indexOf(req), 1);
-          }
+          checkComparisonRequest(req);
         }
       },
     });
     const repeater = E(timer).makeRepeater(0n, quoteInterval);
     return E(repeater).schedule(handler);
   }
-  await startTimer();
+
+  // Only start the ticker if we have actual price changes.
+  if (tradeList) {
```

Comment:
Combine the conditionals rather than have two identical await statement.
```javascript
  if ((tradeList && tradeList.length) > 1 || (priceList && priceList.length > 1)) {
```

Reply:
I did change it to something along these lines.  Let's see if you like.